store/dataStore.ts

```diff
@@ -0,0 +1,17 @@
+import { create } from 'zustand';
+
+interface Example {
+    id: number;
+    name: string;
+}
+interface ExampleStore {
+    examples: Example[];
+}
+
+const dataStore = () => ({
+    examples: [],
+})
+
+const useDataStore = create<ExampleStore>(dataStore)
```

Comment:
That's a great suggestion, but per this document that Shashi mentioned earlier (https://medium.com/globant/react-state-management-b0c81e0cbbf3#7c3b), I modified it to match. Also, I'd prefer to leave them as-is because in my opinion, it's easier to read. Let me know what your thoughts are :)

Reply:
That's fair! This is definitely something that could be argued as more subjective as both are correct ways to go about this.

Reply:
I agree - both ways are correct. (Although I am bias to keeping the format to two lines for readability) :)